src/cosalib/aws.py

```diff
@@ -103,7 +103,9 @@ def aws_run_ore_replicate(build, args):
 
 @retry(reraise=True, stop=stop_after_attempt(3))
 def aws_run_ore(build, args):
-    ore_args = ['ore']
+    # First add the ore command to run before any options
+    ore_args = ['ore', 'aws', 'upload']
+
     if args.log_level:
         ore_args.extend(['--log-level', args.log_level])
```

Comment:
Does this break passing `--log-level` though?

Reply:
seems to work OK

```
$ cosa buildextend-aws --upload     --build=34.20210630.dev.1     --region=us-east-1 --arch=aarch64     --bucket s3://prod-account-match-fcos-builds/ami-import --log-level=DEBUG
[INFO]: symlink is for a build and publish command
[INFO]: ore target aws found via symlink
[INFO]: Targeting architecture: aarch64
[INFO]: Targeting build: 34.20210630.dev.1
[INFO]: Processed build for: Fedora CoreOS testing-devel (FEDORA-COREOS-aarch64) 34.20210630.dev.1
[INFO]: operating on fedora-coreos-34.20210630.dev.1-aws.aarch64.vmdk
[INFO]: executing upload commands for ore
+ ore aws upload --log-level DEBUG --region us-east-1 --bucket s3://prod-account-match-fcos-builds/ami-import --ami-name fedora-coreos-34.20210630.dev.1 --name fedora-coreos-34.20210630.dev.1 --ami-description "Fedora CoreOS testing-devel 34.20210630.dev.1" --file builds/34.20210630.dev.1/aarch64/fedora-coreos-34.20210630.dev.1-aws.aarch64.vmdk --disk-size-inspect --delete-object --arch aarch64
2021-06-30T20:34:08Z cli: Started logging at level DEBUG
```